src/cueTextTracks.js

```diff
@@ -0,0 +1,136 @@
+/**
+* This feature allows metadata text tracks to be manipulated once available, @see process.
+* It also allows ad implementations to leverage ad cues coming through
+* text tracks, @see processAdTrack
+**/
+
+import videojs from 'video.js';
+
+const cueTextTracks = {};
+
+/**
+* This feature allows metadata text tracks to be manipulated once they are available,
+* usually after the 'loadstart' event is observed on the player
+* @param player A reference to a player
+* @param processTrack A method that performs some operations on a metadata text track
+**/
+cueTextTracks.process = function(player, processTrack) {
+  const tracks = player.textTracks();
+  const prepareTrack = function(track) {
+    if (track.kind === 'metadata') {
```

Comment:
Is it possible for there to be metadata tracks we don't want to set the mode of?